src/fermo/app_utils/input_testing.py

```diff
@@ -0,0 +1,236 @@
+        ['MS2Query blank annotation', None],
+        ['MS2Query relative intensity range', None],
+        ['-----', '-----'],
+        ['Process log step', 'Description'],
+        ]
+    return content
+
+
+def create_session_table(
+    params,
+    files,
+    metadata,
+    version,
+    logging
+):
+    '''Parse session file and generate overview table
+
+    Parameters
+    ----------
+    params : `dict`
+    files : `dict`
+    metadata : `dict`
+    version : `dict`
+    logging : `dict`
+
+    Returns
+    -------
+    list of lists: session file info
+    '''
+    content = [
+        ['Date of creation', metadata['date']],
+        ['Time of creation', metadata['time']],
+        ['FERMO version', version],
+        ['-----', '-----'],
+        ['Filename: peaktable', files['peaktable_name']],
+        ['Filename: MS² data', files['mgf_name']],
+        ['Filename: group metadata', files['metadata_name']],
+        ['Filename: quantitative biological data',
+            files['bioactivity_name']],
+        ['Filename: user-library', files['user_library_name']],
+        ['-----', '-----'],
+        ['Mass deviation', params['mass_dev_ppm']],
+        ['Minimum number of fragments per MS² spectrum',
+            params['min_nr_ms2']],
+        ['QuantData factor', params['bioact_fact']],
+        ['Blank factor', params['column_ret_fact']],
+        ['Relative intensity filter range',
+            '-'.join([str(i) for i in params[
+                'relative_intensity_filter_range']])],
+        ['Spectral similarity networking algorithm',
+            params['spec_sim_net_alg']],
+        ['Fragment similarity tolerance', params['spectral_sim_tol']],
+        ['Spectrum similarity score cutoff',
+            params['spec_sim_score_cutoff']],
+        ['Maximum number of links to each feature in spectral network',
+            params['max_nr_links_ss']],
+        ['Minimum number of matched peaks in spectral similarity matching',
+            params['min_nr_matched_peaks']],
+        ['MS2Query used', params['ms2query']],
+        ['MS2Query blank annotation', params['ms2query_blank_annotation']],
+        ['MS2Query relative intensity range',
+            '-'.join([str(i) for i in params['ms2query_filter_range']])],
+        ['-----', '-----'],
+        ['Process log step', 'Description'],
+        ]
+    for entry in logging:
+        content.append([entry, logging[entry]])
+    return content
+
+
+def parse_sessionfile(filename, version):
+    '''Check version compatibility and create session file
+
+    Parameters
+    ----------
+    filename: 'str'
+    version: 'str'; current version of FERMO as read from __version__.py file
+
+    Return
+    ------
+    'dict': containing the parsed file info
+    'str': message for (possible) incompatibility or FileNotFoundError
+    '''
+    try:
+        with open(os.path.join(current_app.config.get('UPLOAD_FOLDER'),
+                               filename)) as f:
```

Comment:
style

Reply:
Also take a look at this handy package: https://docs.python.org/3/library/pathlib.html
Much better to work with than os